### PR TITLE
Fix module exports from Webpack / Browserify

### LINKED ISSUE #16: exported fetch is useless on client side
```
var fetch = require('isomorphic-fetch')
```

doesn't work on client side, because whatwg-fetch module exports nothing.

To make it work `npm-client.js` must look like
```
require('whatwg-fetch');
module.exports = fetch;
```
(instead of `module.exports = require('whatwg-fetch');`)

`client.js` has the same issue, I guess.

### PR DESCRIPTION
**Pull request by robertknight**
In Webpack / Browserify, require('isomorphic-fetch')
returned an empty object because the whatwg-fetch
polyfill only installs fetch() on the global object.

Also correct URLs in package.json

Fixes #16

### CODE REVIEW

**Comment by tracker1** [comment]
+1

**Comment by dendril** [comment]
is there any plan on merge this simple change?

**Comment by matthew-andrews** [comment]
if i merged this change the library would no longer be isomorphic…

happy to accept another PR to fix the URLs in package.json but please keep version as 0.0.0 (i believe it doesn't make sense to store the version number of libs in code in git because that's what tags are for).

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "isomorphic-fetch",
-  "version": "0.0.0",
+  "version": "1.0.0",
```
Comment (matthew-andrews) [inline package.json]: please undo this change

**Comment by matthew-andrews** [comment]
just one small request to update but otherwise this is good to go

**Comment by robertknight** [comment]
> just one small request to update but otherwise this is good to go

Done. Apologies if I missed something that should be obvious but what is the mechanism that sets the npm package version from the Git tag?

**Comment by matthew-andrews** [comment]
@robertknight not so obvious really — little thing i wrote puts the version number in package.json just before running npm publish https://github.com/matthew-andrews/npm-prepublish

just means i can have the same workflow for bower components and npm modules.  (create release on github -> it appears in the npm registry; ie. no command line)
